events/metrics.go

```diff
@@ -0,0 +1,29 @@
+package events
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	MetricLabelGroup       = "group"
+	MetricLabelKind        = "kind"
+	MetricLabelEventType   = "type"
+	MetricLabelEventReason = "reason"
+)
+
+const (
+	MetricNamespace = "operator"
+)
+
+// Cardinality is limited to # objects * # conditions
+var EventCountOpts = prometheus.CounterOpts{
+	Namespace: MetricNamespace,
+	Name:      "event_total",
+	Help:      "The count of events of a given type for an object.",
+}
```

Comment:
Should this be a struct and then we have a function which accepts overrides and returns it?

Reply:
I like that Idea! Updated the to use a common function